ProcessMaker/Http/Controllers/Process/ScreenController.php

```diff
@@ -18,15 +19,12 @@ class ScreenController extends Controller
      */
     public function index()
     {
-        $types = [
-            '' => __('Select Type'),
-            'DISPLAY' => 'Display',
-            'FORM' => 'Form',
-        ];
-
-        if (Script::where('key', 'processmaker-communication-email-send')->exists()) {
-            $types['EMAIL'] = 'Email';
+        $screenTypes = ScreenType::all();
```

Comment:
Use pluck instead https://laravel.com/docs/5.8/collections#method-pluck

Reply:
It is changed to use  method pluck